ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -341,6 +362,12 @@ export default defineComponent({
         getValidation(MhrSectVal.REVIEW_CONFIRM_VALID, MhrCompVal.VALIDATE_STEPS)),
       showEditActions: computed((): boolean => !props.isReadonlyTable),
       homeOwnersTableHeaders: props.isReadonlyTable ? homeOwnersTableHeadersReview : homeOwnersTableHeaders,
+      transferType: computed(() => { return getMhrTransferType.value?.transferType }),
+      showDeathCertificate: computed(() => {
+        // Add other conditions for other transfer Types
+        return localState.transferType === ApiTransferTypes.SURVIVING_JOINT_TENANT ||
+               localState.transferType === ApiTransferTypes.TO_EXECUTOR_PROBATE_WILL
+      }),
```

Comment:
Perfect type of condition/property that would fit into our useTransferOwners composable, since it's specific to a particular transfer owner flow.

Reply:
Agreed